src/pages/Home.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import '../App.css';
+import NewListButton from '../components/NewListButton';
+import * as ls from 'local-storage';
+/*import FirebaseTesting from './FirebaseTest';*/
+import fb from '../lib/firebase';
+import { Link, Redirect } from 'react-router-dom';
+import FooterTabs from '../components/FooterTabs';
+/*import ShoppingList from './ShoppingList';
+import AddItem from './AddItem';*/
+
+function Home() {
+	const [token, setToken] = useState(ls.get('shoppingListToken'));
+	const [inputToken, setInputToken] = useState('');
+	/*const [items, setItems] = useState([]);*/
+	// Adds Value of token entered into inputToken
+	const handleChange = e => {
+		setInputToken(e.target.value);
+	};
+	// Checks the firestore db to see if inputToken exists
+	const checkToken = e => {
+		e.preventDefault();
+		let db = fb.firestore();
+		db.collection(`${inputToken}`)
```

Comment:
Might have to check to see that the input is not empty, because I accidentally clicked the "join list" button without inputting anything, and the firebase threw an error stating inputToken was undefined.